cosmos.go

```diff
@@ -140,6 +143,17 @@ func wsGenerator(wsGenerator websocketGeneratorFun) Option {
 	}
 }
 
+// AutomaticRetries tries to retry failed requests, if appropriate. Retries are limited to maxRetries.
+// Appropriate error codes are 409, 412, 429, 1007, 1008 see https://docs.microsoft.com/en-us/azure/cosmos-db/graph/gremlin-headers#status-codes
+func AutomaticRetries(maxRetries int) Option {
+	return func(c *cosmosImpl) {
+		c.automaticRetries = true
```

Comment:
Wouldn't it make sense to include `c.automaticRetries = true` in the if clause as well?
Because if maxRetries == 0 then automaticRetries == true has no effect

Reply:
then i'd rather remove that flag